File: javascript/bee1018.js
let valor = parseInt(require("fs").readFileSync("stdin", "utf8"));

console.log(valor);
notas = [100, 50, 20, 10, 5, 2, 1];

notas.forEach((nota) => {
    let qntNotas = parseInt(valor / nota);
    console.log(`${qntNotas} nota(s) de R$ ${nota},00`);
    valor -= qntNotas * nota;
});

File: repos/beecrowd/c/bee1018.c
#include <stdio.h>
 
int main() {
    int notas[7] = {100, 50, 20, 10, 5, 2, 1};
    int valor;
    scanf("%d", &valor);
    
    printf("%d\n", valor);
    for(int i = 0; i < 7; i++){
        int qntNotas;
        qntNotas = valor/notas[i]; 
        printf("%d nota(s) de R$ %d,00\n", qntNotas, notas[i]);
        valor %= notas[i];
    }
    return 0;
}